src/include/AMS.h

```diff
@@ -58,9 +58,12 @@ typedef enum { UBALANCED = 0, BALANCED } AMSExecPolicy;
 typedef enum { None = 0, CSV, REDIS, HDF5, RMQ } AMSDBType;
 
 typedef enum {
-  FAISSMean =0,
-  FAISSMax,
-  DeltaUQ // Not supported
+  AMSUQPolicy_BEGIN = 0,
+  FAISS_Mean,
+  FAISS_Max,
+  DeltaUQ_Mean,
+  DeltaUQ_Max,
```

Comment:
I don't like this setup. We probably want a cleaner interface. For the moment we can accept this. Can you add a TODO here.

Reply:
Done